src/hawkbit-client.c

```diff
@@ -231,9 +250,12 @@ static gint rest_request(enum HTTPMethod method, const gchar* url, JsonBuilder*
 {
         gchar *postdata = NULL;
         struct rest_payload fetch_buffer;
-
+        glong http_code = 0;
         CURL *curl = curl_easy_init();
-        if (!curl) return -1;
+        if (!curl) {
+                g_set_error(error, RHU_HAWKBIT_CLIENT_CURL_ERROR, -1, "curl init failed");
```

Comment:
The same above produces the message `"Unable to start libcurl easy session"`. We should make that consistent in any direction.

Reply:
You're right. I'll use "Unable to start libcurl easy session" for both cases.